<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramKeyGrip</class>
 <widget class="QMainWindow" name="ProgramKeyGrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="verticalSlider_green">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:vertical {
	background-color: rgb(0, 0, 0);

    border: 5px solid rgb(0, 255, 0);
    height: 50px;
    margin: -30px -30px;
	border-radius: 30px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Key Grip</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_red">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>299</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:vertical {
	background-color: rgb(255, 255, 255);

    border: 5px solid rgb(200, 0, 0);
    height: 50px;
    margin: -30px -30px;
	border-radius: 30px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_yellow">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>299</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:vertical {
	background-color: rgb(0, 0, 0);

    border: 5px solid rgb(198, 186, 52);
    height: 50px;
    margin: -30px -30px;
	border-radius: 30px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_blue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>299</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:vertical {
	background-color: rgb(255, 255, 255);

    border: 5px solid rgb(0, 0, 255);
    height: 50px;
    margin: -30px -30px;
	border-radius: 30px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trigger</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trigger</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Intensity level specific to Key grip program</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
